src/ServiceControl/Monitoring/SqlDbMonitoringDataStore.cs

```diff
@@ -0,0 +1,126 @@
+﻿namespace ServiceControl.Monitoring
+{
+    using System;
+    using System.Data.SqlClient;
+    using System.Threading.Tasks;
+    using Contracts.Operations;
+    using Dapper;
+    using Infrastructure;
+
+    class SqlDbMonitoringDataStore : IMonitoringDataStore
+    {
+        readonly EndpointInstanceMonitoring monitoring;
+
+        public SqlDbMonitoringDataStore(EndpointInstanceMonitoring monitoring)
+        {
+            this.monitoring = monitoring;
+        }
+
+        public async Task CreateIfNotExists(EndpointDetails endpoint)
+        {
+            var connectionString = Environment.GetEnvironmentVariable("SQLServerConnectionString");
+
```

Comment:
I'm not sure what the standard practice is with this type of thing, but perhaps the sql connection string should be passed in the constructor, the same way the `RavenDbMonitoringDataStore` gets the `IDocumentStore` passed in? If we store the connection string on `Settings` then it will be available at the time of setting up the `HeartbeatMonitoringComponent`